core/src/main/scala/org/apache/spark/sql/execution/oplog/impl/OpLogRdd.scala

```diff
@@ -528,6 +529,19 @@ class OpLogRdd(
   override def compute(split: Partition, context: TaskContext): Iterator[Row] = {
     logDebug(s"starting compute for partition ${split.index} of table $fqtnUpper")
     try {
+      val currentHost = SparkEnv.get.executorId
+      val expectedHost  = bucketHostMap.getOrElse(split.index,"")
+      require(expectedHost.nonEmpty, s"No preferred host found." +
+          s" Error while getting executor host," +
+          s" please check RecoveryService logs for more information.")
```

Comment:
What is a RecoveryService logs. Are we creating a separate log for this mode?

Reply:
We have TraceRecoveryMode flag for debugging. I meant to ask the user to check logs from RecoveryService class, which is visible in the logs. Will improve the statement to be clearer.